README.md

```diff
@@ -1,209 +1,113 @@
-(specified in the authentication process [default: ~/.darwin/projects]).
 
-```python
-from darwin.client import Client
 
-client = Client.local()
-dataset = client.get_remote_dataset(slug="example-dataset")
-dataset.pull()
-```
+---
 
+## Usage as a Command Line Interface (CLI)
 
----
+Once installed, `darwin` is accessible as a command line tool.
+A useful way to navigate the CLI usage is trough the help command `-h/--help` which will 
+provide additional information for each command available. 
 
-### Command line
 
-`darwin` is also accessible as a command line tool.
+### Client Authentication 
 
+To perform remote operations on Darwin you first need to authenticate.
+This requires a [team-specific API-key](https://darwin.v7labs.com/?settings=api-keys).  
+If you do not already have a Darwin account, you can [contact us](https://www.v7labs.com/contact) and we can set one up for you.
 
-#### Client Authentication 
+To start the authentication process, use:
 
-A username (email address) and password is required to authenticate. 
-If you do not already have a Darwin account, register for free at [https://darwin.v7labs.com](https://darwin.v7labs.com).
 ```
 $ darwin authenticate
-Username (email address): simon@v7labs.com
-Password: *******
-Project directory [~/.darwin/projects]: 
-Projects directory created /Users/simon/.darwin/projects
-Authentication succeeded.
 ```
 
+You will be then prompted to insert the API-key, whether you want to set the corresponding team as 
+default and finally the desired location on the local file system for the datasets of that team.
+This process will create a configuration file at `~/.darwin/config.yaml`.
+This file will be updated with future authentications for different teams.
 
-#### Listing local and remote datasets 
+
+### Listing local and remote datasets 
 
 Lists a summary of local existing projects
 ```
-$ darwin local
-NAME                IMAGES     SYNC DATE          SIZE
-example-project          3         today      800.2 kB
+$ darwin dataset local
+NAME            IMAGES     SYNC_DATE         SIZE
+mydataset       112025     yesterday     159.2 GB
 ```
 
 Lists a summary of remote projects accessible by the current user.
 
 ```
-$ darwin remote
-NAME                 IMAGES     PROGRESS     ID
-example-project           3         0.0%     89
+$ darwin dataset remote
+NAME                 IMAGES     PROGRESS
+myteam/mydataset     112025        73.0%
 ```
 
 
-#### Create/remove a dataset 
+### Create/remove a dataset 
 
-Creates an empty dataset remotely.
+To create an empty dataset remotely:
 
 ```
-$ darwin create example-dataset
-Dataset 'example-project' has been created.
-Access at https://darwin.v7labs.com/datasets/example-project
+$ darwin dataset create test
+Dataset 'test' (myteam/test) has been created.
+Access at https://darwin.v7labs.com/datasets/579
 ``` 
 
-To delete the project on the server add the `-r` /`--remote` flag
+Note that in this case `579` will be the dataset ID.
```

Comment:
I think we shouldn't really talk about dataset ids

Reply:
removed